app/models/transaction.rb

```diff
@@ -0,0 +1,76 @@
+class Transaction < ActiveRecord::Base
```

Comment:
Transaction is a poor model name because it conflicts with the sql definition of the word `Transaction`. I'd give this model a more specific name.